eland/tests/field_mappings/test_mappings_with_type_pytest.py

```diff
@@ -22,87 +22,94 @@
 
 
 class TestMappingsWithType(TestData):
-
     def test_mappings_with_type(self):
         # Unless we spin up a 6.x index, this is difficult
         # to test. This is not ideal, but supporting some basic
         # features on 6.x indices makes eland more generally usable.
         #
         # For now, just test function:
-        mapping7x = OrderedDict({
-            "my_index": {
-                "mappings": {
-                    "properties": {
-                        "city": {
-                            "type": "text",
-                            "fields": {
-                                "keyword": {
-                                    "type": "keyword"
-                                }
+        mapping7x = OrderedDict(
```

Comment:
We should be able to remove 'OrderedDict' in this method.

Reply:
Done!